frontend/src/features/admin-form/settings/components/PaymentSettingsSection/PaymentSettingsSection.tsx

```diff
@@ -78,6 +105,15 @@ const BeforeConnectionInstructions = ({
     )
   }, [settings])
 
+  const isSingleSubmission = !!settings?.isSingleSubmission
+
+  const isPaymentsDisabled = isEmailsPresent || isSingleSubmission
+
+  const PaymentsNotAllowedMessageText = getPaymentsNotAllowedMessageText(
```

Comment:
Typo on casing for `Payments...`?
```suggestion
  const paymentsNotAllowedMessageText = getPaymentsNotAllowedMessageText(
```

Reply:
hi @KenLSM, the rationale why i used caps is because the getPaymentsNotAllowedMessageText returns a jsx component, hence i thought to capitalize for any name that references a jsx element

what are your thoughts on this?

Reply:
fixed by refactoring as own component and using `isPaymentsDisabled` instead of non-nullness of component as check of whether to render.